app/src/main/kotlin/com/wire/android/ui/common/UserProfileAvatar.kt

```diff
@@ -0,0 +1,68 @@
+package com.wire.android.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.wire.android.R
+import com.wire.android.ui.common.Dimensions.USER_AVATAR_DEFAULT_SIZE
+
+@Composable
+fun UserProfileAvatar(avatarUrl: String, size: Dp = USER_AVATAR_DEFAULT_SIZE, modifier: Modifier = Modifier, onClick: () -> Unit) {
+    ConstraintLayout(
+        modifier = modifier
+            .padding(10.dp)
+            .size(size)
+    ) {
+        val (avatarImg, statusImg) = createRefs()
+        Image(
+            painter = painterResource(getAvatarAsDrawable(avatarUrl)),
+            contentDescription = "user profile avatar",
```

Comment:
Internationalization. 
Can this be extracted to a `R.string.content_description_user_profile_avatar` ?

Reply:
Totally missed that one. Good eyes 🦅